pkg/api-server/policies_endpoints.go

```diff
@@ -24,6 +24,9 @@ func addPoliciesWsEndpoints(ws *restful.WebService, mode core.CpMode, readOnly b
 				SingularDisplayName: def.SingularDisplayName,
 				PluralDisplayName:   def.PluralDisplayName,
 				IsExperimental:      def.IsExperimental,
+				IsTargetRefBased:    def.IsTargetRefBased,
+				IsInbound:           def.HasFromTargetRef,
```

Comment:
Why the different name?

Reply:
I guess that’s related to what we want to do with this field: show whether a policy type is an inbound and/or outbound one. And I guess this can come from a few different sources one of them being whether is has a from target ref?